src/lvl_script.c

```diff
@@ -6920,6 +7013,10 @@ void script_process_value(unsigned long var_index, unsigned long plr_range_id, l
   struct CreatureModelConfig *crconf;
   struct PlayerInfo *player;
   struct Dungeon *dungeon;
+  struct TrapConfigStats* trapst;
+  struct DoorConfigStats* doorst;
+  struct ManfctrConfig* mconf;
+  struct ManufactureData* manufctr;
```

Comment:
what exactly are these doing here? don't see them used anywhere in the parts you added

Reply:
Not sure how these got there, I'll remove them